rendering_styles/nautical.render.xml

```diff
@@ -557,7 +557,7 @@
 			<case tag="natural" value="crater"/>
 			<case tag="natural" value="hot_spring"/>
 			<case tag="natural" value="geyser"/>
-			<case tag="natural" value="wetland" order="6"/>
```

Comment:
orders should be avoid using at all cause it's not consistent with default osm styles, but here there was exception unclear why

Reply:
But with no order attribute set wetland polygons get (partially) displayed on top of cable areas for example, this does not look good.

Reply:
Z-fighting... on sea there are different rules than on the land, so I'm not sure @xmd5a2